src/game_map.cpp

```diff
@@ -128,6 +128,7 @@ void Game_Map::Dispose(bool is_load_savegame) {
 
 	map.reset();
 	animation.reset();
+	interpreter.reset(new Game_Interpreter_Map(0, true));
```

Comment:
Please edit the issue number this fixes in the commit

Reply:
also you can't simply reset the global Interpreter on map change, events from the previous map continue.

Reply:
Fixed